components/shared/tsconfig.json

```diff
@@ -61,7 +61,7 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
   },
   "include": [
-    "./src/**/*.ts"
+    "**/src/**/*.ts"
```

Comment:
why have we done this change?

Reply:
this was a remnant of before I added Lerna. removing

Reply:
added back. build failed when we didn't have this. possibly because there are two src files now since we package shared and stream-provider in dist